frontend/src/views/web/challenge/my-submission.html

```diff
@@ -66,7 +66,7 @@ <h6>My Participated Team: {{challenge.participated_team_name}}</h6>
                             <th class="fs-18 w-300" ng-if="challenge.isChallengeHost &&
                                     !challenge.isStaticCodeUploadChallenge" data-field="file">Environment Logs</th>
                             <th class="align-center fs-18 w-300" data-field="file">Submitted at</th>
-                            <th class="align-center fs-18 w-300" ng-if="challenge.currentPhaseLeaderboardPublic"
+                            <th class="align-center fs-18 w-300" ng-if="!challenge.disable_private_submission && challenge.currentPhaseLeaderboardPublic"
```

Comment:
Why do we need this check in `my-submissions` page? My submissions should ideally display all the submissions for a user.